src/scss/base/reset/p-toast.scss

```diff
@@ -23,5 +23,16 @@ body .p-toast .p-toast-message {
 
   &.p-toast-message-error {
     background-color: $wharf_status_red !important;
+
+    a,
+    button {
+      margin-right: 0.8em;
+    }
+
+    a:hover {
+      text-decoration: none;
+      background: #546e7a;
+      color: #fff;
```

Comment:
This would probably be part of a larger css rework but usually it would be good to link magic numbers such as color to scss theme variables such that the theme can be centrally controlled.

Reply:
Good point.
Extracting these to `variables.scss` before they get forgotten is probably the best way to go, what do you think?

Reply:
The root scss files are specified here- https://github.com/iver-wharf/wharf-web/blob/df795e9358f8cba66f27878c7668269e0a52ecd7/angular.json#L29-L32

I think the natural thing to do would be to link the root file at `%repo_root%/styles.scss` for style that we intend to keep global. Then i would define specific themes in the `%repo_root%/scss/**/*` and use scss import declarations to import them whenever nessecary. Note the global file could also import specific or default themes from the `%repo_root%/scss/**/*`.This should probably be up for discussion though.